static/src/stylesheets/module/commercial/glabs/_hosted-gallery.scss

```diff
@@ -62,10 +66,15 @@
     .hosted-gallery__image-sizer {
         position: absolute;
         overflow: hidden;
+        width: 100%;
+        height: 100%;
     }
     .u-responsive-ratio {
         position: static;
     }
+    img {
+        object-fit: cover;
```

Comment:
Ben is cropping the images in the grid, so they will hopefully be the right size and won't get stretched! Just have to update the URLs next